code/src/Components/HappyThought.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import moment from "moment" // this is to format the date
+import "./happyThought.css"
+
+export const HappyThought = props => {
+  const { message, hearts, createdAt, _id } = props.thought
+
+  const handleClick = () => {
+    fetch(`https://technigo-thoughts.herokuapp.com/${_id}/like`, {
+      method: "POST",
+      body: "",
+      headers: { "Content-Type": "application/json" }
+    }).then(() => props.onLiked(_id))
+  }
+
+  return (
+    <article className='happy-thought'>
+      <h3>{message}</h3>
+      <div className="happyThoughtFooter">
+        <p className="buttonWrapper">
```

Comment:
Oh, here you have changed the way you write the html. Here you use `"` and in `HappyForm` you are using `'`. Please stick to one way of writing the syntax in the project